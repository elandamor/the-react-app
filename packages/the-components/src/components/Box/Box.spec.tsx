import { render } from '@testing-library/react';
import React from 'react';
import { Box } from './index';

describe('Box', () => {
  it('renders without crashing', () => {
    const { container } = render(<Box />);
    expect(container.firstChild?.nodeName.toLowerCase()).toBe('div');
    expect(container.children).toHaveLength(1);
    expect(container.firstChild).toMatchSnapshot();
    expect(container.firstChild).toHaveStyle(`
      display: flex;
      flex-direction: column;
    `);
  });
});
