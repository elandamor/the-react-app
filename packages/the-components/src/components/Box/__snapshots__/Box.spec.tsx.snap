// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Box renders without crashing 1`] = `
.emotion-0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  position: relative;
  width: 100%;
  box-sizing: border-box;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

<div
  class="emotion-0"
/>
`;
